README.md

```diff
@@ -76,8 +76,9 @@ Import { Chat } from 'botframework-webchat';
 
 const YourApp = () => {
     <div>
+        <link href="https://unpkg.com/botframework-webchat/botchat.css" rel="stylesheet" />
```

Comment:
When I followed the instructions, I `npm installed`'ed WebChat; however, there was no styling by default.  Perhaps a different line needs to be added to include the styling from WebChat.  Can you provide that, and I'll test and update the PR.